semantic_release/cli.py

```diff
@@ -259,28 +261,28 @@ def publish(**kwargs):
         # Get config options for uploads
         dist_path = config.get("dist_path")
         remove_dist = config.get("remove_dist")
-        upload_pypi = config.get("upload_to_pypi")
-        upload_to_pypi_glob_patterns = config.get("upload_to_pypi_glob_patterns")
+        upload_to_artifact_repository = config.get("upload_to_repository") and config.get("upload_to_pypi")
```

Comment:
Shouldn't this be an `or`?

Reply:
These default to true, using `and` means only one option needs to be disabled.

Reply:
This PR depreciates `upload_to_pypi` renaming it to `upload_to_repository`, so they are essentially the same option. The custom repository will be set to PyPI by default.

Reply:
Ah, I *am* overlooking something. Thanks for explaining!